### ISSUE TITLE
EmptyLinesAroundArguments with inline private definition

### ISSUE THREAD

**Issue by lacrosse** [bbatsov/rubocop#5458]
```ruby
class EmptyLinesTest
  private def method
    a = 2

    a + 2
  end
end
```

## Expected behavior

```
Inspecting 1 file
.

1 file inspected, no offenses detected
```

## Actual behavior

```
Inspecting 1 file
C

Offenses:

empty_lines_test.rb:6:1: C: Layout/EmptyLinesAroundArguments: Empty line detected around arguments.

1 file inspected, 1 offense detected
```

## RuboCop version

0.52.1 (using Parser 2.4.0.2, running on ruby 2.5.0 x86_64-darwin16)

**Comment by garettarrowood** [comment]
This is indeed broken on `0.52.1`.  But it's been fixed in our unreleased version with #5340 .  See issue #5333 for this example.

**Comment by lacrosse** [comment]
Thanks for pointing out!
